pyproject.toml

```diff
@@ -89,7 +90,8 @@ target-version = [
   "py39",
   "py310",
   "py311",
-  "py312"
+  "py312",
+  "py313"
 ]
```

Comment:
py313 here but python3.13 is blocked above?

Reply:
This sections specifically covers the coding conventions. It just indicates that we should also be following rules in Python3.13.